### PR TITLE
added info about params related to meeting_url in RTMP

### PR DESCRIPTION
**Pull request by Nixon-Samuel**

### CODE REVIEW

File: docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx [hunk 1]
```diff
@@ -73,11 +73,13 @@ curl \
...
 
-There are also a few additional query params which can be used for the 100ms app link -
+Please check the following to understand how these params are useful while starting RTMP streaming or browser recording with 100ms app links:
 
--   `auth_token=token1234`: this can be used to pass auth token generated on your side to the recording web-app
--   `ui_mode=activespeaker`: this can be used to open active speaker mode by default for recording. In active speaker mode,
-    the current speaker will show up in center, and rest of the peers will show up in the sidebar.
+-   `skip_preview=true`: skips the preview page and directly joins the meeting with header and controls disabled.
+-   `skip_preview_headful=true`: skips the preview page and directly joins the meeting with header and controls enabled.
+-   `auth_token=token1234`: Use this to pass auth token generated on your side to the recording web-app
```
Comment (arjunattam) [inline docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx:80]: how does this work?

File: docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx [hunk 2]
```diff
@@ -73,11 +73,13 @@ curl \
...
+-   `ui_mode=activespeaker`: opens active speaker mode by default for recording. In active speaker mode, the current speaker will show up in the center, and rest of the peers will show up in the sidebar.
+-   'name=johndoe': Use this to add a name for the respective peer.
```
Comment (arjunattam) [inline docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx:82]: these quotes are different? not backticks?

File: docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx (hunk 1 above)
Comment (Nixon-Samuel) [reply docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx]: updated it.

File: docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx (hunk 2 above)
Comment (Nixon-Samuel) [reply docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx]: fixed

File: docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx [hunk 3]
```diff
@@ -73,11 +73,13 @@ curl \
...
+Please check the following to understand how these params are useful while starting RTMP streaming or browser recording with 100ms app links:
 
--   `auth_token=token1234`: this can be used to pass auth token generated on your side to the recording web-app
--   `ui_mode=activespeaker`: this can be used to open active speaker mode by default for recording. In active speaker mode,
-    the current speaker will show up in center, and rest of the peers will show up in the sidebar.
+-   `skip_preview=true`: skips the preview page and directly joins the meeting with header and controls disabled.
+-   `skip_preview_headful=true`: skips the preview page and directly joins the meeting with header and controls enabled.
+-   `auth_token=token1234`: Use this to pass the auth token to authenticate the room join request for the beam bot to join the web app to start recording.
+-   `ui_mode=activespeaker`: opens active speaker mode by default for recording. In active speaker mode, the current speaker will show up in the center, and rest of the peers will show up in the sidebar.
+-   `name=johndoe`: Use this to add a name for the respective peer.
```
Comment (arjunattam) [inline docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx:82]: why will i use this for beam?

**Review by arjunattam — APPROVED** [review]

**Pull request merged**